index.html

```diff
@@ -3,9 +3,58 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel='stylesheet' href='style/index.css'>
     <title>Document</title>
+
 </head>
-<body>
-    
+<body>    
+ <div class="container">
+    <header>
+<h1> Brandon Gunn </h1>
+<nav>
+    <a href=>Home</a> 
```

Comment:
and dont forget to link "home" to its corresponding page as well per MVP (:
href="home.html"